tests/unit_tests/fn/test_extents.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <gridtools/fn/extents.hpp>
+
+#include <type_traits>
+
+#include <gtest/gtest.h>
+
+#include <gridtools/common/integral_constant.hpp>
+#include <gridtools/common/tuple_util.hpp>
+
+namespace gridtools::fn {
+    namespace {
+        using namespace literals;
+        struct a {};
+        struct b {};
+        struct c {};
+
+        static_assert(is_extent<extent<a, 0, 0>>::value);
+        static_assert(is_extent<extent<a, -1, 1>>::value);
+
+        namespace {
```

Comment:
why is this unnamed namespace?

Reply:
that's a leftover, it was a check of the `extent` concept. Removed...